src/containers/BuySellCrypto/PaymentMethod/SendTransaction/SendTransaction.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+    View,
+    Text,
+    ScrollView,
+  } from 'react-native';
+import { NavigationActions } from 'react-navigation';
+import { sendTransaction } from '../../../../actions/actions/PaymentMethod/WyreAccount';
+import Button1 from '../../../../symbols/button1';
+import styles from './SendTransaction.styles';
+import {
+    getTransactions
+} from '../../../../actions/actions/PaymentMethod/WyreAccount';
+import {
+    selectTransactionHistoryIsFetching,
+    selectTransactionHistory
+} from '../../../../selectors/paymentMethods';
+import { ListItem } from "react-native-elements";
+
+class SendTransaction extends Component {
+    constructor(props) {
+        super(props);
+        const {
+            paymentMethod,
+            fromCurr,
+            fromVal,
+            toCurr,
+            toVal,
+            buy
+        } = this.props.navigation.state.params;
+        this.state = {
+            paymentMethod,
```

Comment:
You don't need this data in the state. It will not change, use the data from the props. No need for the constructor override